webserver/script/migrations/0001_initial.py

```diff
@@ -260,7 +261,7 @@ class Migration(migrations.Migration):
             ],
             options={
                 'db_table': 'script_algorithm_ev_load_forecast',
-                'unique_together': {('config',)},
+                'unique_together': {('config', 'controlled')},
```

Comment:
Ideally you don't edit existing migrations, you generate a new one. If you re-run migrations against the DB these changes won't be reflected unless you delete the migration table in the DB.

Reply:
You're right